ethexe/contracts/script/Deployment.s.sol

```diff
@@ -34,6 +36,7 @@ contract DeploymentScript is Script {
 
         address mirrorAddress = vm.computeCreateAddress(deployerAddress, vm.getNonce(deployerAddress) + 2);
         address mirrorProxyAddress = vm.computeCreateAddress(deployerAddress, vm.getNonce(deployerAddress) + 3);
+        address middlewareAddress = vm.computeCreateAddress(deployerAddress, vm.getNonce(deployerAddress) + 4);
```

Comment:
`vm.computeCreateAddress(deployerAddress, deployerNonce)` is used to compute the future contract address. after `mirrorProxy = new MirrorProxy(address(router))` you need to create `middleware` to make this address valid.